mitmproxy/addons/dumper.py

```diff
@@ -124,6 +125,9 @@ def _echo_request_line(self, flow):
             url = flow.request.pretty_url
         else:
             url = flow.request.url
+        terminalWidthLimit = shutil.get_terminal_size()[0] - 25
```

Comment:
How about a simple `max(shutil.get_terminal_size()[0] - 25, 50)` ? This way we can assure that we have at least 50 characters of the URL. 😃